### ISSUE TITLE
Error in `cna_report_segments_nondiploid.Rmd`

### ISSUE THREAD

**Issue by vladsaveliev** [sigven/pcgr#30]
Sigve,

I'm getting this error when running PCGR with the cnv file attached:
[pcgr_cna_test.zip](https://github.com/sigven/pcgr/files/1974109/pcgr_cna_test.zip)

```
./pcgr.py --input_cna cup__cup_tissue-somatic.tsv . grch37 pcgr_configuration_somatic.toml cup__cup_tissue-somatic
```

The output tail:

```
018-05-04 20:49:54 [INFO] Detected 0 segments subject to amplification/deletion (0 deletions, 0 gains according to user-defined log(2) ratio thresholds)
2018-05-04 20:49:54 [INFO] Detected 0 proto-oncogenes subject to amplification (log(2) ratio >= 0.7
2018-05-04 20:49:54 [INFO] Detected 0 tumor suppressor genes subject to homozygous deletion (log(2) ratio <= -1.1
2018-05-04 20:49:54 [INFO] Looking up SCNA biomarkers for precision oncology - any tumortype
2018-05-04 20:49:54 [INFO] ------
2018-05-04 20:49:54 [INFO] Assigning elements to PCGR value boxes
Quitting from lines 30-40 (./cna_report_segments_nondiploid.Rmd)
Quitting from lines NA-72 (./cna_report_segments_nondiploid.Rmd)
Error in .subset2(public_bind_env, "initialize")(...) :
  object 'cna_segments' not found
Calls: <Anonymous> ... withVisible -> eval -> eval -> <Anonymous> -> <Anonymous>
Execution halted
```

It must be because none of the events were identified as important and `cna_segments` were undefined before passing to `crosstalk::SharedData$new(cna_segments)` in the `else` block here? 
https://github.com/sigven/pcgr/blob/master/src/R/pcgrr/inst/templates/cna_report_segments_nondiploid.Rmd

Or am I doing something wrong you think?

Vlad

**Comment by sigven** [comment]
Likely a bug, let me have a look tonight. Thanks so much for your testing, very valuable for me.

**Comment by sigven** [comment]
Should be fixed now (0.6.2)
